features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/event/TimelineItemVoiceView.kt

```diff
@@ -172,6 +186,36 @@ private fun RetryButton(
     }
 }
 
+@Composable
+private fun PlaybackSpeedButton(
+    speed: Float,
+    onClick: () -> Unit,
+) {
+    val speedText = when (speed) {
+        0.5f -> "0.5×"
+        1.0f -> "1×"
+        1.5f -> "1.5×"
+        2.0f -> "2×"
+        else -> "${speed}×"
+    }
+    androidx.compose.foundation.layout.Box(
+        modifier = Modifier
+            .background(
+                color = ElementTheme.colors.bgCanvasDefault,
+                shape = RoundedCornerShape(12.dp)
+            )
+            .padding(horizontal = 8.dp, vertical = 4.dp)
+            .clickable(onClick = onClick),
```

Comment:
The clickable modifier should be above the padding, so that the clickable area is bigger.